Foundation/NSXMLElement.swift

```diff
@@ -49,8 +49,15 @@ open class XMLElement: XMLNode {
         @method initWithXMLString:error:
         @abstract Returns an element created from a string. Parse errors are collected in <tt>error</tt>.
     */
-    public init(xmlString string: String) throws {
-        NSUnimplemented()
+    public convenience init(xmlString string: String) throws {
+        // If we prepend the XML line to the string
+        let docString = """
+        <?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\"?>\(string)
```

Comment:
Mostly for my own education on the new triple-quote syntax: do we still need to backslash escape the quotes?